kubernetes/cache/cache.go

```diff
@@ -28,6 +28,10 @@ const (
 	kialiCacheWaypointsKey   = "waypoints"
 )
 
+var (
+	klog = log.WithGroup("kialiCache")
+)
```

Comment:
This is just personal preference but this could also be a variable on the cache/kubecache structs rather than a global variable. This is really nit picky but I'd rather we keep `log` than `klog`. You can always change the import and keep the log statements the same: `klog "github.com/kiali/kiali/log"`.

Reply:
> I'd rather we keep log than klog

I agree - and I tried to do that. I couldn't figure out a good way to do it and have the code compile-able :)

Putting the logger in some struct could be a good way to do it.